ivy/src/main/scala/sbt/librarymanagement/ivy/UpdateOptions.scala

```diff
@@ -109,7 +109,7 @@ object UpdateOptions {
       interProjectFirst = true,
       latestSnapshots = true,
       cachedResolution = false,
-      gigahorse = true,
+      gigahorse = sys.props.get("sbt.gigahorse.enabled") map { _.toLowerCase == "true" } getOrElse true,
```

Comment:
I think "enabled" is redundant. `sbt.gigahorse`?

Reply:
I considered that there might be further gigahorse-related flags in the future. Hence `gigahorse.enabled`, `gigahorse.xxx`, `gigahorse.yyy`,... etc

Reply:
Not really likely, and even if it happens I think it's fine to have `sbt.gigahorse` and `sbt.gigahorse.followredirect` ect.